plugins/nf-google/src/main/nextflow/cloud/google/batch/GoogleBatchTaskHandler.groovy

```diff
@@ -201,6 +201,8 @@ class GoogleBatchTaskHandler extends TaskHandler {
                     .addNetworkInterfaces(networkInterface)
             )
 
+        allocationPolicy.putAllLabels(task.config?.getResourceLabels()  ?: Collections.emptyMap() as Map<String,String>)
```

Comment:
```suggestion
        allocationPolicy.putAllLabels(task.config.getResourceLabels())
```

the `config`must exist, for tests it's easy to mock it (it also returns an empty map if not defined)